whois-nrtm4/src/main/java/net/ripe/db/nrtm4/NrtmFileRepo.java

```diff
@@ -0,0 +1,48 @@
+package net.ripe.db.nrtm4;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+
+
+@Service
+public class NrtmFileRepo {
+
+    private final String path;
+
+    NrtmFileRepo(
+        @Value("${nrtm.file.path:/tmp}") final String path
+    ) {
+        this.path = path;
+    }
+
+    boolean checkIfFileExists(final String name) {
+        return new File(path, name).exists();
```

Comment:
Can we put access modifier?

Reply:
it's not needed so i left it out because in general it's a good idea to encapsulate functionality at package level. i'll make it public if you prefer.

Reply:
No, Its cover in my later comments, I think we should use FileHelper in whois-commons. Thanks